ros/2/src/openvslam/src/run_slam.cc

```diff
@@ -116,8 +119,8 @@ void tracking(const std::shared_ptr<openvslam::config>& cfg, const std::string&
     if (track_times.size()) {
         std::sort(track_times.begin(), track_times.end());
         const auto total_track_time = std::accumulate(track_times.begin(), track_times.end(), 0.0);
-        std::cout << "median tracking time: " << track_times.at(track_times.size() / 2) << "[s]" << std::endl;
-        std::cout << "mean tracking time: " << total_track_time / track_times.size() << "[s]" << std::endl;
+        RCLCPP_INFO(ros->node_->get_logger(),"Median tracking time: %f [s] ", track_times.at(track_times.size() / 2));
```

Comment:
I'd make these debug messages, not sure they're really important for ROS users to see, seems more interesting for evaluation testing than for actual deployed use.

Reply:
ok!